manpages/brew-cask.1

```diff
@@ -162,6 +162,10 @@ Target location for Preference Panes\. The default value is \fB~/Library/Prefere
 Target location for QuickLook Plugins\. The default value is \fB~/Library/QuickLook\fR\.
 .
 .TP
+\fB\-\-dictionarydir=<path>\fR
```

Comment:
Don't edit this file directly. Make the change in [brew-cask.1.md](https://github.com/Homebrew/brew/blob/8995d8b9e5ccaf3c56aceff655c51f924737a11e/Library/Homebrew/manpages/brew-cask.1.md), then run `brew man` to generate the manpage.